src/main/java/database/Database.java

```diff
@@ -0,0 +1,142 @@
+package database;
+
+import java.io.*;
+import java.util.HashMap;
+
```

Comment:
A javadoc describing the database might be necessary.

Reply:
Sure I'll add it.